docs/Building-an-OpenSearch-and-OpenSearch-Dashboards-Distribution.md

```diff
@@ -12,12 +12,18 @@ Manifests are the single source of truth for building and testing the distributi
   
 ### Building from Source
   
-Input manifests are the source of building the distributions. Each input manifest also contains the docker image that can be used to build these distributions and avoid installing bunch of packages. See [CI image details](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/3.0.0/opensearch-3.0.0.yml#L8-L9) in the manifest.
+Input manifests are the source of building the distributions. Each input manifest also contains the docker image that can be used to build these distributions and avoid installing bunch of packages. See [CI image details](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/3.0.0/opensearch-3.0.0.yml#L8-L9) in the manifest along with java version used to build the distribution.
   
 To run locally:
 ```bash
-  docker run -it -d --entrypoint bash opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2 -e JAVA_HOME=/opt/java/openjdk-20
+  docker run -it --entrypoint bash opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1
```

Comment:
If you dont add -d then container will stop running after exit.

Reply:
That's okay. Currently devs not familiar with the -d get confused since you need to run `docker exec` to go into it.